### PR TITLE
Work through rubocop errors

### PR DESCRIPTION
**Pull request by despo**

### CODE REVIEW

**Comment by despo** [comment]
Related to #635

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -23,15 +23,15 @@ def index
...
+    event = Event.find_by(slug: params[:id])
 
     @event = EventPresenter.new(event)
     @host_address = AddressPresenter.new(@event.venue.address)
 
     if logged_in?
       invitation = Invitation.find_by(member: current_user, event: event, attending: true)
       if invitation
-        redirect_to event_invitation_path(@event, invitation) and return
+        redirect_to event_invitation_path(@event, invitation) && return
```
Comment (EGiataganas) [inline app/controllers/events_controller.rb]: I think this should be

```ruby
return redirect_to event_invitation_path(@event, invitation)
```

Can you please double check if there is a spec for that?

File: app/controllers/feedback_controller.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@ def show
 
     if feedback_request.nil?
       flash[:notice] = I18n.t('messages.feedback_not_found')
-      redirect_to root_path and return
+      return redirect_to root_path
```
Comment (EGiataganas) [inline app/controllers/feedback_controller.rb]: 👍

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (EGiataganas) [inline app/controllers/events_controller.rb]: I think this should be

```ruby
return redirect_to event_invitation_path(@event, invitation)
```

Can you please double check if there is a spec for that?

Comment (despo) [reply app/controllers/events_controller.rb]: Good catch. I'll take a look.

**Pull request merged**